esmvalcore/cmor/check.py

```diff
@@ -579,6 +579,9 @@ def _check_time_coord(self):
                     self.report_error(msg, var_name, freq)
                     break
 
+        if hasattr(coord, "attributes"):
+            coord.attributes = None
```

Comment:
Hm. I think even an empty cube will have a `iris._cube_coord_common.LimitedAttributeDict` here. Maybe something like `coord.attributes.clear()` would be more appropriate? If one really wants to be so callous as to always destroy all the metadata.

Reply:
but `coord.attributes = None` still preserves the `attributes` and it makes it an empty dict `{}` so I don't see why use `clear()` - would that be safer?

Reply:
Do we really want to remove everything from all posible datasets?. I prefer to do 
```
coord.attributes.pop('time_origin', None)
```
This attribute looks redundant with the units, so it's unlikely to lose relevant information, but this may not be the case with other ones in other datasets

Reply:
@valeriupredoi, you are right. I personally find `.clear()` clearer.

Regardless, the `if` line can go since all `coord`s have the `attributes` attribute and I am with @jvegasbsc in that I'd prefer only to remove the offender.

Reply:
Yeah I just remembered about attributes: positive: up/down for the Z coord that's really needed but note this is done only for the T coord - anything we want to keep from the time coords attrs? I think we should just nuke them all just so we don't run into this again - up to yous, it's all the same for me

Reply:
There could be. For example for climatological time axis. Let's do what we must and keep what we can.

Reply:
good stuff, gents, cheers for the comments - Ive changed it so we remove just `time_origin` but kept the check on actual existence of attrs member, just in case - you know, stranger things may happen :beer:

Reply:
Like what? :thinking: But seriously, that's part of the api and we should avoid unnecessary bloat.

Reply:
ok, boss, removed the if statement :beer: